include/wx/uilocale.h

```diff
@@ -14,6 +14,7 @@
 
 #if wxUSE_INTL
 
+#include "wx/arrstr.h"
```

Comment:
Minor, but we're already using `wxVector<wxString>` in this header/class, let's use it instead of `wxArrayString` for the new functions too.